phpBB/language/en/acp/styles.php

```diff
@@ -54,6 +54,7 @@
 	'INSTALL_STYLES'							=> 'Install styles',
 	'INSTALL_STYLES_EXPLAIN'					=> 'Here you can install new styles.<br>If you cannot find a specific style in list below, check to make sure style is already installed. If it is not installed, check if it was uploaded correctly.',
 	'INVALID_STYLE_ID'							=> 'Invalid style ID.',
+	'INVALID_STYLE_MESSAGE'						=> 'The directory <strong><i>%1$s</i></strong> does not contain a valid style.',
```

Comment:
Perhaps quote the directory, as done in other language strings: `“%s”`

Reply:
I think it's usually either in bold or with quotes, not both at the same time. Do you think using quotes would be better here instead of using bold text?